go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -493,47 +610,15 @@ func (n *NodeController) EnsureHybridOverlayBridge(node *kapi.Node) error {
 			"IN_PORT",
 			extVXLANName, subnet.String(), hotypes.HybridOverlayVNI, n.drMAC.String(), portMACRaw))
 
-	// Add a route via the hybrid overlay port IP through the management port
-	// interface for each hybrid overlay cluster subnet
 	mgmtPortLink, err := util.GetNetLinkOps().LinkByName(types.K8sMgmtIntfName)
 	if err != nil {
 		return fmt.Errorf("failed to lookup link %s: %v", types.K8sMgmtIntfName, err)
 	}
+
+	err = n.createOrReplaceRoutes(mgmtPortLink, nil)
 	mgmtPortMAC := mgmtPortLink.Attrs().HardwareAddr
```

Comment:
@JacobTanenbaum can you move this mgmtPortMAC line right underneath the if err != nil bit ?

Reply:
done